### PR TITLE
Fix CSVWireTest. Deprecate unused methods.

### LINKED ISSUE #804: Fix CSVWireTest

### PR DESCRIPTION
**Pull request by yevgenp**
Closes #804

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/wire/WireType.java [hunk 1]
```diff
@@ -600,47 +599,6 @@ public boolean tryAdvance(@NotNull Consumer<? super T> action) {
                 }, false);
     }
 
-    @NotNull
-    public <T> Map<String, T> fromFileAsMap(String filename, @NotNull Class<T> tClass) throws IOException, InvalidMarshallableException {
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/wire/WireType.java:604]: should deprecate before removing

File: src/test/java/net/openhft/chronicle/wire/CSVWireTest.java [hunk 2]
```diff
@@ -19,15 +19,10 @@
...
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.Test;
 
-import java.io.IOException;
-import java.util.Map;
-
 import static org.junit.Assert.*;
 
-@Ignore("TODO FIX")
```
Comment (JerryShea) [inline src/test/java/net/openhft/chronicle/wire/CSVWireTest.java:30]: nice

File: src/test/java/net/openhft/chronicle/wire/CSVWireTest.java [hunk 3]
```diff
@@ -62,33 +59,6 @@ public void tstFrom2() {
         doTestWire(wire);
     }
 
-    @Test
-    public void tstFrom3() throws IOException {
```
Comment (JerryShea) [inline src/test/java/net/openhft/chronicle/wire/CSVWireTest.java:66]: why remove?

File: src/main/java/net/openhft/chronicle/wire/WireType.java (hunk 1 above)
Comment (yevgenp) [reply src/main/java/net/openhft/chronicle/wire/WireType.java:604]: This method is not used anywhere (across all Chronicle repos) but in the following removed test. 
IMO it does not make sense to spend more time to fix it (I've tried) as it's a dead code.

Comment (JerryShea) [reply src/main/java/net/openhft/chronicle/wire/WireType.java:604]: OK. @peter-lawrey 's decision.... I am OK to remove although if we want to preserve API compatibility we could deprecate but replace the contents with `throw new UnsupportedOperationException()`

**Comment by peter-lawrey** [comment]
Unless we know a feature is broken, we should delay removal for 2 versions. i.e. removal in x.27.

**Comment by yevgenp** [comment]
> Unless we know a feature is broken, we should delay removal for 2 versions. i.e. removal in x.27.

Like I've mentioned above that method is not used anywhere (across all Chronicle repos) but in the following removed test.
It'd require quite an effort to make it work. Which makes no sense IMO.
Do you suggest to deprecate it?

File: src/main/java/net/openhft/chronicle/wire/WireType.java (hunk 1 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/wire/WireType.java:604]: I wouldn't throw UOE; it's better to fail at compile time.

Comment (JerryShea) [reply src/main/java/net/openhft/chronicle/wire/WireType.java:604]: so @peter-lawrey you would rather remove the methods?

Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/wire/WireType.java:604]: All to be deprecated for removal in x.27

**Comment by yevgenp** [comment]
Removed broken method and it's test. Deprecated two others.

**Review by peter-lawrey — APPROVED** [review]

**Pull request merged**